platform/config-core/src/main/java/org/motechproject/config/core/service/impl/CoreConfigurationServiceImpl.java

```diff
@@ -101,14 +101,33 @@ public void saveBootstrapConfig(BootstrapConfig bootstrapConfig) {
 
     @Override
     public ConfigLocation getConfigLocation() {
-        Iterable<ConfigLocation> configLocations = configLocationFileStore.getAll();
+        return getLocation(true);
+    }
+
+    @Override
+    public ConfigLocation getMotechLocation() {
```

Comment:
We should keep the config location separate from the new directory location. The only thing tying them together should be the value of the default config location, based on the motech.dir, not user.home.